### DISCUSSION TITLE
RuntimeError: CreateModel failed with 'Error reading the proto buffer model file.' (0x3005)

### DISCUSSION THREAD

**Discussion by framilano** [coqui-ai/STT#2010] [Q&A] [answered]
Hi everyone, trying to use the pretrained models on Ubuntu 20.04 and Python3.8. Installed stt with pip3, but when I'm executing the stt command;
```
stt --model model.tflite  --parser coqui-stt-0.9.3-models.scorer --audio test.wav
```

I get:

```
Loading model from file model.tflite
TensorFlow: v2.3.0-6-g23ad988fcde
 Coqui STT: v0.10.0-alpha.10-0-g9b517632
Warning: reading entire model file into memory. Transform model file into an mmapped graph to reduce heap usage.
2021-11-01 22:13:59.377042: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
Data loss: Can't parse English_STT_v0.9.3/model.tflite as binary proto
Traceback (most recent call last):
  File "/usr/local/bin/stt", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/stt/client.py", line 148, in main
    ds = Model(args.model)
  File "/usr/local/lib/python3.8/dist-packages/stt/__init__.py", line 40, in __init__
    raise RuntimeError(
RuntimeError: CreateModel failed with 'Error reading the proto buffer model file.' (0x3005)
```

I'm running it in a Docker Container.

**Comment by framilano** [comment, marked as answer]
I solved installing from the same requirements available in the python websocket example.
I installed these modules with pip

```
setuptools~=45.2.0
deepspeech~=0.8.2
black>=20.8b1
ffmpeg-python==0.2.0
sanic==20.3.0
pyhocon==0.3.54
numpy~=1.17.0
```

**Reply by lissyx** [reply]
> I solved installing from the same requirements available in the python websocket example.

This is likely wrong, you are pulling `deepspeech` binary and maybe out of luck it is able to load the model, but this is not what you want. Start from a clean virtualenv and properly install latest `stt` python wheel.

**Reply by framilano** [reply]
I tried installing the latest STT, but I had no luck.

**Reply by reuben** [reply]
The error message you shared is not present in the latest version, so either the installation is not working or your somehow using an installation from a different environment.

**Comment by lissyx** [comment]
> Data loss: Can't parse English_STT_v0.9.3/model.tflite as binary proto

You're trying to read a TFLite model with ProtocolBuffer code.

**Reply by framilano** [reply]
I just followed this example, it does the same thing
https://stt.readthedocs.io/en/latest/DEPLOYMENT.html#using-the-python-package

**Comment by reuben** [comment]
It sounds like you're using an older version of the package. If you upgrade to STT==1.0.0 it should work fine.
